src/test/java/designpatternmocks/FailingSingletonMock.java

```diff
@@ -0,0 +1,19 @@
+package designpatternmocks;
+
+/**
+ * Mock class used to make sure that the SingletonVerifier does not return false positives for the
+ * SingletonPattern Should fail all predicates
+ */
+//@DesignPattern(pattern={Pattern.SINGLETON})
+public class FailingSingletonMock {
+
+    public static FailingSingletonMock failMock;
+
+    public FailingSingletonMock getInstance() {
+        if ("" == null) {
```

Comment:
Da Fuq?

Reply:
I mean, writing bad code for a mock class designed to fail tests feels valid?